Mastermind/Presentation/InputOutput/ConsoleInputOutput.cs

```diff
@@ -0,0 +1,121 @@
+using Mastermind.Domain.BusinessRules;
+using Mastermind.Domain.Models;
+
+namespace Mastermind.Presentation.InputOutput
+{
+    public class ConsoleInputOutput : IInputOutput
+    {
+        private static readonly string[] AllColours = { 
+            ColourSquares.Red,
+            ColourSquares.Blue,
+            ColourSquares.Green,
+            ColourSquares.Orange,
+            ColourSquares.Purple,
+            ColourSquares.Yellow
+        };
+
+        private const char CharToIntConversion = '0';
+
+        public PlayerInput GetAGuessInput()
+        {
+            PrintGetUserGuessMessage();
+            return GetTheUserGuessedColours();
+        }
+
+        public void OutputWelcomeMessage()
+        {
+            PrintOutput(ConsoleMessages.Title);
+            PrintOutput(ConsoleMessages.GameInformation);
+        }
+
+        public void OutputColourArray(string[] colours)
```

Comment:
Super nitpicky, but do we need the data type (Array) in the name? Would `OutputColours` suffice?